crates/driver/openapi.yml

```diff
@@ -333,41 +333,51 @@ components:
     SolveResponse:
       description: |
         Response of the solve endpoint.
-      type: array
-      items:
-        type: object
-        properties:
-          solutionId:
-            description: |
-              The unique identifier of the solution.
-
-              This id is used to identify the solution when executing it.
-            type: integer
-          score:
-            description: |
-              The objective value of the solution.
-            type: number
-          submissionAddress:
-            description: |
-              The address that will be used to submit the solution.
-            $ref: "#/components/schemas/Address"
-          orders:
-            description: executed orders
-            type: array
-            items:
-              type: object
-              description: |
-                Mapping of order uid to net executed amount (including all fees).
-              additionalProperties:
-                $ref: "#/components/schemas/TokenAmount"
-          clearingPrices:
-            description: |
-              Mapping of hex token address to price.
-
-              The prices of tokens for settled user orders as passed to the settlement contract.
+      type: object
+      properties:
+        solutions:
+          type: array
+          items:
             type: object
-            additionalProperties:
-              $ref: "#/components/schemas/BigUint"
+            properties:
+              solutionId:
+                description: |
+                  The unique identifier of the solution.
+    
+                  This id is used to identify the solution when executing it.
+                type: string
+                example: "123"
+              score:
+                description: |
+                  The objective value of the solution.
+                type: string
+                example: "123"
+              submissionAddress:
+                description: |
+                  The address that will be used to submit the solution.
+                $ref: "#/components/schemas/Address"
+              orders:
+                description: executed orders
+                type: array
+                items:
```

Comment:
`orders` is an object mapping a `OrderUid` to executed amounts.

Reply:
fixed